styles.css

```diff
@@ -23,132 +24,61 @@ article.latest-guess,
 div.row {
   flex-direction: row;
   justify-content: space-between;
-/*   align-items: flex-end; */
 }
 
 main {
   align-items: flex-start;
 }
 
 main {
-  background-color: darkgray;
+  background-color: #d0d2d3;
 }
```

Comment:
There are two main selectors.  You can combine these two to one selector so that it can save space.

Reply:
Oops! Good catch!